desktop/core/src/desktop/js/apps/editor/components/result/reactExample/ReactExample.tsx

```diff
@@ -76,11 +79,17 @@ const ReactExample: FunctionComponent<ReactExampleProps> = ({ activeExecutable,
     // Also make sure that the component specific Antd style is imported in the file
     // 'root-wrapped-antd.less'.
     <React.StrictMode>
-      <div className="react-example antd">
+      <div className="react-example cuix antd">
```

Comment:
is cuix and antd classes compulsory? How will someone new coming into the project know to add this.

Reply:
Sorry my previous answered was never saved when I resolved the issue. Yes, but only if the component is a react root. I've updated the contribute frontend instructions.